server/utils/mail-template/mail-template.util.js

```diff
@@ -0,0 +1,118 @@
+                              <tr>
+                                <td align="center"> <span style="color:#48545d;font-size:22px; line-height: 24px;">
+                                    ${emailTitle}
+                                  </span>
+                                </td>
+          
+                              </tr>
+                              
+                              <tr>
+                                <td align="center" style="margin:50px;"> <span style="color:#48545d; font-size:14px; line-height:20px;">
+                                    ${message}
+                                  </span>
+                                </td>
+                              </tr>
+                              <tr>
+                                <td height="20"></td>
+                              </tr>
+                              <tr>
+                                <td valign="top" width="48%" align="center"> <span>
+                                    <a href="${url}" style="display:block; padding:15px 25px; background-color:#40617B; color:#ffffff; border-radius:3px; text-decoration:none;">Visit Authors Haven</a>
+                                  </span>
+                                </td>
+                              </tr>
+                            </tbody>
+                          </table>
+                        </td>
+                      </tr>
+                      <tr>
+                        <td height="60"></td>
+                      </tr>
+                    </tbody>
+                  </table>
+                  <!-- Main CONTENT end here -->
+                  <!-- FOOTER start here -->
+                  <table width="100%" cellpadding="0" cellspacing="0" border="0">
+                    <tbody>
+                      <tr style="background-color: #40617B; color:white">
+                        <td valign="top" align="center"> <span style="font-family: -apple-system,BlinkMacSystemFont,sans-serif; color:#9EB0C9; font-size:10px;">&copy;
+                            <a href="#" target="_blank" style="color:#9EB0C9; text-decoration:none;">Author's
+                              Haven</a>
+                            &nbsp;&nbsp;|&nbsp;&nbsp;
+                            <a href="#" target="_self" style="color:#9EB0C9; text-decoration:none;">&copy; 2019 </a>
+                            &nbsp; &nbsp; &nbsp; <a href="#" style="color:#9EB0C9; text-decoration: none;">click to unsuscribe for this message</a>
+                          </span>
+                        </td>
+                      </tr>
+                      <tr>
+                        <td height="50">&nbsp;</td>
+                      </tr>
+                    </tbody>
+                  </table>
+                  <!-- FOOTER end here -->
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</body>
+</html>`;
```

Comment:
Good work man, just curious why did you use tables for your arrangements instead of div blocks

Reply:
I feel it is important to find a universal approach, because most email clients and rendering engines support the use of tables across. 

Reply:
Ok then, Good Job